### ISSUE TITLE
Group collision methods not working as expected

### ISSUE THREAD

**Issue by calebfoss** [quinton-ashley/p5play#240]
I can't seem to get any of Group's collision methods to work.

Here's what I tried 
[view on web editor](https://editor.p5js.org/cfoss/sketches/aGOBqZuBQ)
```JavaScript
let group;
let spriteInGroup;
let other;

function setup() {
  new Canvas(400, 400);
  group = new Group();
  spriteInGroup = new Sprite(200, 200, 100, 100, "static");
  group.add(spriteInGroup);
  other = new Sprite(300, 200, 100, 100, "dynamic");
  other.addSpeed(1, 180);
}

function draw() {
  background(220);
  console.log(
    `FRAME: ${frameCount}
group.colliding(other): ${group.colliding(other)}
group.collides(other): ${group.collides(other)}
group.collided(other) :${group.collided(other)}
spriteInGroup.colliding(other): ${spriteInGroup.colliding(other)}
spriteInGroup.collides(other): ${spriteInGroup.collides(other)}
spriteInGroup.collided(other): ${spriteInGroup.collided(other)}
other.colliding(group): ${other.colliding(group)}
other.collides(group: ${other.collides(group)}
other.collided(group): ${other.collided(group)}`
  );
  noLoop();
}
function mousePressed() {
  redraw();
}
```
group's collision methods return 0, false, false respectively every frame.

Am I misunderstanding how to use these methods, or is this a bug?

**Comment by quinton-ashley** [comment]
Yes that's a bug, it only works if you create the sprite with the group constructor right now but yeah it'd be good if sprites added to the group could be included in its collision detection. I will try fixing this.

**Comment by quinton-ashley** [comment]
I found the problem and fixed it! Will release an update soon. Thank you for making this test, this was a big problem, I guess I had not tested the group functions as throughly as I thought.

**Comment by quinton-ashley** [comment]
fixed in v3.6.8
